packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -57,4 +65,24 @@ describe('packages/MongoMenu', () => {
 
     expect(onLogout).toHaveBeenCalledTimes(1);
   });
+
+  test('renders the account link as a disabled button when set to the empty string', () => {
+    renderedComponent.rerender(
+      <MongoMenu
+        user={{ name: 'Leafy', email: 'leafy@mongodb.com' }}
+        activeProduct={Product.Atlas}
```

Comment:
I don't expect people to directly use the enum in most cases, so a better way to test this might be to explicitly write the string out here (just `'atlas'` I believe). That way, this test will break if we make a breaking API change.

Reply:
Additionally (and correct me if I'm wrong), but you're setting props here that aren't required or being tested at this point. You *should* be fine omitting `onLogout`, and `onProjectChange` at the very least.

Reply:
This block was more or less copy-pasted and just moved, but I can definitely make these changes.

Reply:
Both of these were already provided in the tests prior to my changes, but I'm happy to clean them up now.

Reply:
All of these were how the test was written before I jumped in, though I can definitely leaf off `onLogout` and `onProjectChange` when I rerender here. I have no problem just replacing the usage of `Product` here.